components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -2267,24 +2267,42 @@ def _annotations(request, objtype, objid, conn=None, **kwargs):
     query += """
```

Comment:
OK, that wasn't my experience at all. I could only load this is the user's default group was set to the group where the data was. @will-moore are you testing with 5.2?